### PR TITLE
Update Polkadot

### PR DESCRIPTION
**Pull request by Doordashcon**
This PR adds the constant value `EXISTENTIAL_DEPOSIT` to polkadot, kusama & westend runtime constants as needed to update cumulus.

### CODE REVIEW

File: runtime/westend/constants/src/lib.rs [hunk 1]
```diff
@@ -20,6 +20,9 @@
 pub mod currency {
 	use primitives::v0::Balance;
 
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
```
Comment (bkchr) [inline runtime/westend/constants/src/lib.rs:23]:
```suggestion
	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
```

**Review by bkchr — APPROVED** [review]

**Review by bkchr — CHANGES_REQUESTED** [review]
This value should also be used in the runtimes directly.

File: runtime/polkadot/constants/src/lib.rs [hunk 2]
```diff
@@ -22,6 +22,9 @@ pub use self::currency::DOLLARS;
 pub mod currency {
 	use primitives::v0::Balance;
 
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9010).
+	pub const EXISTENTIAL_DEPOSIT: Balance = 10 * CENTS;
```
Comment (bkchr) [inline runtime/polkadot/constants/src/lib.rs:26]: These values are also not correct and the docs are also not correct.

**Comment by Doordashcon** [comment]
> This value should also be used in the runtimes directly.

should have noticed that earlier, TY

File: runtime/polkadot/constants/src/lib.rs (hunk 2 above)
Comment (Doordashcon) [reply runtime/polkadot/constants/src/lib.rs]: used the values from cumulus, updated now

**Review by bkchr — APPROVED** [review]

**Pull request merged**